lib/ts/recipe/authRecipe/authWidgetWrapper.tsx

```diff
@@ -77,14 +72,14 @@ const Redirector = <T, S, R, N extends NormalisedConfig<T | GetRedirectionURLCon
                     void Session.getInstanceOrThrow()
                         .validateGlobalClaimsAndHandleSuccessRedirection(
                             {
-                                rid: props.authRecipe.config.recipeId,
-                                successRedirectContext: {
-                                    action: "SUCCESS",
-                                    isNewRecipeUser: false,
-                                    isNewPrimaryUser: false,
-                                    redirectToPath: getRedirectToPathFromURL(),
-                                },
+                                action: "SUCCESS",
+                                isNewRecipeUser: false,
+                                isNewPrimaryUser: false,
+                                isFirstFactor: true,
```

Comment:
what if this is called during post login? Like if someone wants to link a social login to their existing session?